styleguide/source/_patterns/01-molecules/06-components/01-pull-quote/_pull-quote.scss

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file pull-quote.scss
+ *
+ * Copyright 2017 Palantir.net, Inc.
+ */
+
+.pull-quote {
+  border-left: 12px solid $red;
+  padding-left: 10px;
+  margin: 30px 0;
+  font-style: normal;
+
+  @include breakpoint($bp-small) {
+    margin: 40px 0;
+    padding-left: 15px;
+    border-left: 16px solid $red;
+  }
+}
+
+.pull-quote_quote {
+  @extend %b2;
+  margin-bottom: 6px;
+
+  @include breakpoint($bp-small) {
+    margin-bottom: 9px;
+  }
+}
+
+.link-pull-quote {
```

Comment:
@aCyborg the link font weight looks a bit light. I think it's suppose to be semibold.... though when I tried changing it in the inspector it didn't update to a heavier weight. I'm not sure we have specific font weight changes from `400` to `500`.